AnwesenheitsSimulation/module.php

```diff
@@ -217,7 +217,7 @@ public function UpdateData()
 
         $simulationData = $this->GetDataArray(array_merge($weekDays, $singleDays), $targetIDs);
         if (count($simulationData) == 0) {
-            $this->SetValue('SimulationDay', 'Zu wenig Daten!');
+            SetValue($this->GetIDForIdent('SimulationDay'), $this->Translate('Not enough data!'));
```

Comment:
Hier wird das $this->SetValue wieder auf das reguläre SetValue geändert, bitte mit $this->SetValue